File: src/main/java/fr/polyconseil/smartcity/tefpsclient/dto/tv/ParkingRightDTO.java
package fr.polyconseil.smartcity.tefpsclient.dto.tv;

public class ParkingRightDTO extends ParkingRightCreationDTO {

    private String ticketId;

    public String getTicketId() {
        return ticketId;
    }

    public void setTicketId(String ticketId) {
        this.ticketId = ticketId;
    }
}

File: php5.6/dto/tv/ParkingRightDTO.php
<?php

namespace TefpsClientBundle\dto\tv;

use TefpsClientBundle\dto\tv\ParkingRightCreationDTO;

use JMS\Serializer\Annotation\Type;
use JMS\Serializer\Annotation\SerializedName;

class ParkingRightDTO extends ParkingRightCreationDTO
{
    /**
     * @Type("string")
     * @SerializedName("ticketId")
     */
    private $ticketId;

    public function getTicketId() {
        return $this->ticketId;
    }

    public function setTicketId($ticketId) {
        $this->ticketId = $ticketId;
    }
}
